05-history.md

```diff
@@ -202,6 +202,25 @@ on the other hand,
 moving backward and forward in time becomes much easier.
 
 
+
+> ## Checking out changed files {.challenge}
+>
+> Assume you have started to work on something, like above and git says to discard thing.
```

Comment:
Suggested rewording:  `Suppose that you have changed a working file under version control, but have not yet added or committed those changes.  If you now run the command `git checkout -- mars.txt`, what actions does Git take?`